cmd/spec/main.go

```diff
@@ -53,6 +53,9 @@ func main() {
 	gen.addSchema("v1.ResponseError", &payloads.ResponseError{})
 	gen.addSchema("v1.Source", &payloads.SourceResponse{})
 	gen.addSchema("v1.InstanceType", &payloads.InstanceTypeResponse{})
+	gen.addSchema("v1.Reservation", &payloads.ReservationResponse{})
+	gen.addSchema("v1.NoopReservation", &payloads.NoopReservationResponse{})
+	gen.addSchema("v1.AWSReservation", &payloads.AWSReservationResponse{})
```

Comment:
I see one flaw with this code, those v1 names do not include Request or Response words. For some endpoints we need to differentiate between the two in the specification. Like in this case. This needs to be fixed, let us do it only for reservations in this PR tho:

```
    gen.addSchema("v1.ReservationResponse", &payloads.ReservationResponse{})
	gen.addSchema("v1.NoopReservationRequest", &payloads.NoopReservationRequest{})
	gen.addSchema("v1.NoopReservationResponse", &payloads.NoopReservationResponse{})
    // the same for AWS
```

Reply:
FYI I am filing a draft to refactor the existing ones: https://github.com/RHEnVision/provisioning-backend/pull/167